CMSgov/design-system
# [WNMGDS-2921] Implement Filter Chip Web Component

## Summary

- [Ticket](https://jira.cms.gov/browse/WNMGDS-2921)
- Addition of `ds-filter-chip` web component
- Stories and Unit tests added

## How to test

1. Pull down this branch locally
2. Run `yarn build`
3. Run `yarn test:unit:wc` to confirm that unit tests pass
4. Run `yarn storybook` & confirm that all attributes are editable in the single filter chip example. I've disabled the props table in the multichip example because the controls didn't do anything.
5. Also in Storybook confirm that a `ds-delete` custom event is fired when clicking the chip
6. I've not added an example in Astro, but have in the CDN Web Components. You can confirm the functionality there.

## Checklist
